server/bell_oauth_profile.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var log = require('./logger')('server.bell.profile');
+var db = require('./db/db');
+var config = require('./config');
+
+// this is the custom provider profile function used by Bell to allow us
+// to convert oauth tokens into profile information.
+function profile (credentials, params, get, profileCb) {
```

Comment:
We could, although it doesn't matter a lot to me. If you feel strongly about it, let me know, and I'll change it here. Otherwise, we can file an issue and followup later.